File: src/lib/scala/Slate.Core/src/main/scala/slate/core/mods/ModuleInfo.scala
/**
  * <slate_header>
  * author: Kishore Reddy
  * url: https://github.com/kishorereddy/scala-slate
  * copyright: 2016 Kishore Reddy
  * license: https://github.com/kishorereddy/scala-slate/blob/master/LICENSE.md
  * desc: a scala micro-framework
  * usage: Please refer to license on github for more info.
  * </slate_header>
  */
package slate.core.mods

case class ModuleInfo(
                    name         : String,
                    desc         : String,
                    version      : String,
                    isInstalled  : Boolean,
                    isEnabled    : Boolean,
                    isDbDependent: Boolean,
                    totalModels  : Int,
                    source       : String,
                    dependencies : String,
                    models       : Option[List[String]] = None
                 )
{

}

File: src/support/integration/src/main/kotlin/kiit/integration/mods/ModuleInfo.kt
/**
 *  <kiit_header>
 * url: www.kiit.dev
 * git: www.github.com/slatekit/kiit
 * org: www.codehelix.co
 * author: Kishore Reddy
 * copyright: 2016 CodeHelix Solutions Inc.
 * license: refer to website and/or github
 * 
 * 
 *  </kiit_header>
 */

package kiit.integration.mods

data class ModuleInfo(
    val name: String,
    val desc: String,
    val version: String,
    val isInstalled: Boolean,
    val isEnabled: Boolean,
    val isDbDependent: Boolean,
    val totalModels: Int,
    val source: String,
    val dependencies: String,
    val models: List<String>? = null
)